02/PrintRecords.cpp

```diff
@@ -0,0 +1,30 @@
+#include <fstream>
+#include <iostream>
+#include <string>
+#include "PrintRecords.h"
+
+using namespace std;
+
+namespace samples
+{
+	void ManageRecordsExample()
+	{
+		Record info;
+		ofstream fileStream("studentRecords.dat", ios_base::out);
+
+		while (true)
+		{
+			cout << "First name: ", cin >> info.FirstName;
+			cout << "Last name: ", cin >> info.LastName;
+			cout << "Student ID: ", cin >> info.StudentID;
+			cout << "Score: ", cin >> info.Score;
+
+			fileStream.seekp(0);
+			fileStream << info.FirstName << " " << info.LastName << " "
```

Comment:
so this one does
1. read from cin
2. print to filestream.

i want you to add more functionalities here.
1. when the program starts, it loads the file and print all records in nice format
2. then it asks if you want to a) add another student or b) modify existing
 a) prompts another student to add and print the full file contents again
 b) prompts which student (by order, 1,2,3,4,5) to edit and print the full file content after modified

Still current ManageRecords() function should be the entry point. In this function you will call other functions to make this works. I would assume you would have

PrintRecords(), InputRecord() function at least.